examples/torch/common/export.py

```diff
@@ -8,29 +8,66 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from pathlib import Path
+
 import torch
 
+from examples.common.sample_config import SampleConfig
+from examples.torch.common.example_logger import logger
 from nncf.api.compression import CompressionAlgorithmController
 from nncf.torch.exporter import count_tensors
 from nncf.torch.exporter import generate_input_names_list
 from nncf.torch.exporter import get_export_args
 
 
-def export_model(ctrl: CompressionAlgorithmController, save_path: str, no_strip_on_export: bool) -> None:
+def export_model(ctrl: CompressionAlgorithmController, config: SampleConfig) -> None:
     """
-    Export compressed model. Supported only 'onnx' format.
+    Export compressed model ot OpenVINO format.
 
     :param controller: The compression controller.
-    :param save_path: Path to save onnx file.
-    :param no_strip_on_export: Set to skip strip model before export.
+    :param config: The sample config.
     """
-
-    model = ctrl.model if no_strip_on_export else ctrl.strip()
-
+    model = ctrl.model if config.no_strip_on_export else ctrl.strip()
     model = model.eval().cpu()
 
     export_args = get_export_args(model, device="cpu")
     input_names = generate_input_names_list(count_tensors(export_args))
 
-    with torch.no_grad():
-        torch.onnx.export(model, export_args, save_path, input_names=input_names)
+    input_tensor_list = []
+    input_shape_list = []
+    for info in model.nncf.input_infos.elements:
+        input_shape = tuple([1] + info.shape[1:])
+        input_tensor_list.append(torch.rand(input_shape))
+        input_shape_list.append(input_shape)
+
+    if len(input_tensor_list) == 1:
+        input_tensor_list = input_tensor_list[0]
+        input_shape_list = input_shape_list[0]
+
+    model_path = Path(config.export_model_path)
+    model_path.parent.mkdir(exist_ok=True, parents=True)
+    extension = model_path.suffix
+
+    if extension == ".onnx":
+        with torch.no_grad():
+            torch.onnx.export(model, input_tensor_list, model_path, input_names=input_names)
+    elif extension == ".xml":
+        import openvino as ov
+        from openvino.tools.mo import convert_model
+
+        if config.export_to_ir_via_onnx:
+            model_onnx_path = model_path.with_suffix(".onnx")
+            with torch.no_grad():
+                torch.onnx.export(model, input_tensor_list, model_onnx_path, input_names=input_names)
+            ov_model = convert_model(model_onnx_path, compress_to_fp16=False)
```

Comment:
Why is compression to the FP16 precision was disabled?

Reply:
Removed it, ci-maintenance/job/pt_e2e_tests/job/NNCF/job/nightly/job/TriggerBetta/75/